packages/api-explorer/package.json

```diff
@@ -4,14 +4,16 @@
   "version": "6.1.1",
   "main": "dist/index.js",
   "dependencies": {
-    "@json-editor/json-editor": "^1.4.0-beta.0",
+    "@apidevtools/json-schema-ref-parser": "^9.0.1",
+    "@fortawesome/fontawesome-free": "^5.13.0",
+    "@json-editor/json-editor": "2.2.1",
     "@mia-platform/markdown": "^6.1.0",
     "@mia-platform/oas-extensions": "^6.1.0",
     "@mia-platform/syntax-highlighter": "^6.1.0",
     "@mia-platform/variable": "^6.1.0",
     "choices.js": "^9.0.1",
     "classnames": "^2.2.5",
-    "fetch-har": "^2.0.0",
+    "fetch-har": "2.2.0",
```

Comment:
After the PR gets merged let's open an issue in the project to remember us that we shall resolve the breaking change introduced with the version 2.2.1 patch and update it

Reply:
https://github.com/APIDevTools/json-schema-ref-parser/issues/174

Reply:
fetch-har is not json-schema-ref-parser

Reply:
https://github.com/mia-platform/api-explorer/issues/51